qt/aqt/editor.py

```diff
@@ -1622,10 +1623,13 @@ def _add_image_menu(self, webview: AnkiWebView, menu: QMenu) -> None:
         a = menu.addAction(tr.editing_copy_image())
         qconnect(a.triggered, self.on_copy_image)
 
+        url = webview.lastContextMenuRequest().mediaUrl()
```

Comment:
> The code at present is incorrect, and will print an error message when there's no media URL on the clipboard.

Due to this code?

I can't think of a case where there won't be a media URL because this code runs only when the user right-clicks on an image.

https://github.com/ankitects/anki/blob/81c30bf99e14da7b8d39083583f9c6a853cf4658/qt/aqt/editor.py#L1609

Reply:
lastContextMenuRequest() is returning None there. Happens on two different machines I tested with.

Reply:
Because this PR is built on top of the code added by @bpnguyen107 in https://github.com/ankitects/anki/pull/3362, I would like to ask him if he has any suggestion.